pkg/gofr/crud_handlers.go

```diff
@@ -61,7 +63,16 @@ type entity struct {
 
 // scanEntity extracts entity information for CRUD operations.
 func scanEntity(object interface{}) (*entity, error) {
-	entityType := reflect.TypeOf(object).Elem()
+	if object == nil {
+		return nil, errObjectIsNil
+	}
+
+	objType := reflect.TypeOf(object)
+	if objType.Kind() != reflect.Ptr {
+		return nil, fmt.Errorf("failed to register routes for %s struct, as it is passed by value should be pointer", objType.Name())
```

Comment:
we can just mention that passed object is not pointer, no need to specify `passed by value`
also, we can present struct in quotes like: `'%s' struct`

Reply:
@srinivas365 - The linter error is yet to be fixed.